examples/proxy/broadcast.py

```diff
@@ -42,22 +42,32 @@ def master(group_name: str, worker_num: int, is_immediate: bool = False):
                         you can do something with high priority before receiving replied messages from peers.
             Sync Mode: It will block until the proxy returns all the replied messages.
     """
-    proxy = Proxy(group_name=group_name,
-                  component_type="master",
-                  expected_peers={"worker": worker_num})
+    proxy = Proxy(
+        group_name=group_name,
+        component_type="master",
+        expected_peers={"worker": worker_num}
+    )
 
     if is_immediate:
-        session_ids = proxy.ibroadcast(tag="INC",
-                                       session_type=SessionType.NOTIFICATION)
+        session_ids = proxy.ibroadcast(
+            component_type="worker",
+            tag="INC",
+            session_type=SessionType.NOTIFICATION
+        )
         # do some tasks with higher priority here.
```

Comment:
Start with a capital letter, end with a period.

Same to other comments and docstrings.